Using TensorFlow backend.
Traceback (most recent call last):
  File "prediction.py", line 408, in <module>
    main(config)
  File "prediction.py", line 389, in main
    seed=int(time.time() * 1000) % (2 ** 32 - 1))
  File "prediction.py", line 310, in data_preparing
    training_x['s92'] = [eval(x)[2] for x in training_x['s9']]
  File "prediction.py", line 310, in <listcomp>
    training_x['s92'] = [eval(x)[2] for x in training_x['s9']]
IndexError: list index out of range
Time: 2020-10-12 23:50:06.389206
[Configuration]
'est_config' : ====== Intrinsic Dim Estimation of real dataset ======
'log_dir' : exp/realcase/log
'log_filename' : est.log
'logrs' : -4:0:20

-----------------chromium----------------------
        bugID  s1  s2  s3   s4  s5  s6  s8  s90  s91
0      350007   1   1   0   24   1   0   1    1    0
1      350007   4   3   1   71   2   2   3    0    1
2      350007   1   1   0   31   3   3   1    0    1
3      350007   2   1   1   14   4   7   1    0    1
4      350007   5   3   2   31   5   8   2    0    1
...       ...  ..  ..  ..  ...  ..  ..  ..  ...  ...
69996  526025   4   2   2  118   3   1   3    0    1
69997  526025   1   0   1    5   4   2   1    0    1
69998  526072   2   1   1   12   1   0   2    0    1
69999  526072   1   0   1    6   2   1   1    0    1
70000  526072   1   1   0   44   3   4   1    0    1

[70001 rows x 10 columns]
In training set:
number of target is 8269
number of non-target is 61732
ratio of target is 0.11812688390165854
>> creating intrinsic dimension solver
>> solving...
ues L1 norm to compute the distance
candidate estiamted instrinsic dim: [ 1.45012951  1.47154218  1.44735983  1.50873206  1.61080364  1.85861922
  2.00945499  2.00239938  1.99532968  2.09197567  2.18371638  2.31535863
  2.27238217  2.19980535  2.07771238  1.91650505  1.71707645  1.48025626
  1.19941034 -0.        ]
>> task finished
-----------------eclipse----------------------
